src/main/java/core/basesyntax/FruitShop.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ConverterService;
+import core.basesyntax.service.ProcessData;
+import core.basesyntax.service.ReaderService;
+import core.basesyntax.service.WriterService;
+import core.basesyntax.serviceimpl.ConverterServiceImpl;
+import core.basesyntax.serviceimpl.ProcessDataimpl;
+import core.basesyntax.serviceimpl.ReaderServiceImpl;
+import core.basesyntax.serviceimpl.ReportService;
+import core.basesyntax.serviceimpl.WriterServiceImpl;
+
+public class FruitShop {
+    public static void main(String[] args) {
+        ReaderService readerService = new ReaderServiceImpl();
+        readerService.read("src/main/resources/filename.csv");
```

Comment:
make "src/main/resources/filename.csv" constant
and you should use result List<String> from reading